docs/wallet-integration-guide/src/exchange-integration/workflows.rst

```diff
@@ -3,6 +3,12 @@
 Integration Workflows
 =====================
 
+.. contents::
+   :local:
+   :depth: 2
+   :backlinks: none
```

Comment:
added as https://docs.digitalasset.com/integrate/devnet/exchange-integration/workflows.html only shows the top-level sections in the right side-bar